build/charts/antrea-windows/conf/antrea-agent.conf

```diff
@@ -54,10 +54,9 @@ featureGates:
 # <HOST> to <Service namespace>/<Service name>. For example,
 # "flow-aggregator/flow-aggregator" can be provided to connect to the Antrea
 # Flow Aggregator Service.
-# If PORT is empty, we default to 4739, the standard IPFIX port.
-# If no PROTO is given, we consider "tls" as default. We support "tls", "tcp" and
-# "udp" protocols. "tls" is used for securing communication between flow exporter and
-# flow aggregator.
+# If PORT is empty, we default to 4739. If no PROTO is given, we consider "tls" as default.
+# We support "grpc", "tls", "tcp" and "udp" protocols. "grpc", "tls" are used for securing
+# communication between flow exporter and flow aggregator.
```

Comment:
@antoninbas do we support FlowExport on Windows? I updated a few descriptions accordingly, but I recall we don't support it on Windows?